src/types/form/ReimbursementAccountForm.ts

```diff
@@ -21,6 +21,7 @@ const INPUT_IDS = {
         CITY: 'requestorAddressCity',
         STATE: 'requestorAddressState',
         ZIP_CODE: 'requestorAddressZipCode',
+        IS_ONFIDO_SETUP_COMPLETE: 'isOnfidoSetupComplete',
```

Comment:
Why do we need this new input id here?

Reply:
It turned out we use it here https://github.com/software-mansion-labs/expensify-app-fork/blob/401bb219c74fb615704895554b6876eb25601a11/src/pages/ReimbursementAccount/RequestorOnfidoStep.js#L49 and missed it before